dsl/src/main/groovy/org/cristalise/dsl/persistency/outcome/DynamicForms.groovy

```diff
@@ -29,6 +29,7 @@ import groovy.transform.CompileStatic;
 
 @CompileStatic
 class DynamicForms {
+    String accept = null
```

Comment:
Rename to something more descriptive like `acceptFileTypes` or `acceptMimeTypes` (not sure which one is the correct one).